site/docs/blogs.md

```diff
@@ -28,16 +28,19 @@ Here is a list of company blogs that talk about Iceberg. The blogs are ordered f
 
 **Author**: [Alex Merced](https://www.linkedin.com/in/alexmerced/)
 
+<!-- markdown-link-check-disable-next-line -->
```

Comment:
good point, I should have done that but didn't realize such a setting is available

Reply:
I was thinking about a workflow. When new blogs are added in a PR, it can be placed before `<!-- markdown-link-check-disable -->`. When the link check passes, we place `<!-- markdown-link-check-disable -->` at the beginning again before merging the PR. That's a bit complex, though.